src/config.ml

```diff
@@ -567,6 +586,11 @@ module Conf_map = struct
           | `Tcp (Some `Client) -> "-client"
           | `Tcp (Some `Server) -> "-server"
           | `Tcp None | `Udp -> "")
+    | Protocol_flags, flags ->
+        (* XXX: We assume [flags] is always non-empty *)
+        p () "protocol-flags %a"
+          Fmt.(list ~sep:(any " ") (of_to_string Protocol_flag.to_string))
+          flags
```

Comment:
There are no guarantees flags is not empty. However, a trailing space is not the end of the world...